ios/RNModules/RNLogging/RNLogging/RNLoggingModule.m

```diff
@@ -45,29 +31,61 @@ + (BOOL)requiresMainQueueSetup {
     return YES;
 }
 
-- (id<RNLogger>)logger {
-    if (!_logger) {
-        return [[RNDummyLogger alloc] init];
-    }
-    return _logger;
-}
-
-RCT_EXPORT_METHOD(ancillaryDisplayed:(NSString *)type  provider:(NSString *)provider) {
+RCT_EXPORT_METHOD(ancillaryDisplayed:(NSString *)type provider:(NSString *)provider) {
     [self.logger ancillaryDisplayed:type provider:provider];
 }
 
 RCT_EXPORT_METHOD(ancillaryPurchased:(NSString *)type provider:(NSString *)provider) {
     [self.logger ancillaryPurchased:type provider:provider];
 }
 
+RCT_EXPORT_METHOD(hotelsResultsDisplayed:(id)additionalInfo) {
+    [self.logger hotelsResultsDisplayed:additionalInfo];
+}
+
+RCT_EXPORT_METHOD(hotelsSelectedFilterTag:(NSString *)filterTag) {
```

Comment:
From the spreadsheet as well, it seems the method is only concerned with whether a filter tag was set or not, but I agree that adding the name of the filter tag could be interesting to log. We should discuss this further.

Reply:
Yeah, this is the only thing I didn't do it. I think, we should log this and I'll talk about it with others at Monday.

Reply:
As I posted to slack, the filter tag parameter was added.